skrf/calibration/tests/test_calibration.py

```diff
@@ -1229,16 +1229,10 @@ def setUp(self):
             wg.short(nports=2, name='short'),
             wg.open(nports=2, name='open'),
             wg.match(nports=2, name='load'),
-            None,
-            ]
-        actuals = [
-            wg.short(nports=2, name='short'),
-            wg.open(nports=2, name='open'),
-            wg.match(nports=2, name='load'),
-            wg.thru(),
+            wg.random(n_ports=2),
             ]
 
-        measured = [ self.measure(k) for k in actuals]
+        measured = [ self.measure(k) for k in ideals]
```

Comment:
This test previously tested that without specifying an ideal thru a flush thru is assumed. There isn't now a test for it and it's now the only difference between SOLT and TwelveTerm. I think this could be reverted. In `MultiportSOLTTest` it should be possible to replace thru with`wg.line` though.

Reply:
done.

`SOLTTest `now tests again a flush thru if no ideal thru is given. `MultiportSOLTTest `tests against non-flush thru mismatch**line**mismatch

Reply:
great!